apps/web-giddh/src/app/accounting/journal-voucher/voucher/voucher.component.ts

```
@@ -2113,19 +2516,14 @@ export class AccountAsVoucherComponent implements OnInit, OnDestroy, AfterViewIn
      */
-    public onItemSelected(event: IOption): void {
-        setTimeout(() => {
-            this.currentSelectedValue = '';
-            this.showLedgerAccountList = false;
-            this.changeTab('enter', 'account', true);
-        }, 100);
+    public onItemSelected(event: any): void {
+        this.showLedgerAccountList = false;
+            
         if (event?.value === 'createnewitem') {
             return this.addNewAccount();
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)